src/main/scala/co/ledger/wallet/daemon/modules/DaemonCacheModule.scala

```
@@ -32,21 +32,16 @@ object DaemonCacheModule extends TwitterModule {
     val poolsService = injector.instance[PoolsService](classOf[PoolsService])
 
     def synchronizationTask(): Unit = {
-      val t0 = System.currentTimeMillis()
-      Try(Await.result(poolsService.syncOperations, 5.minutes)) match {
-        case Success(result) =>
-          result.foreach { r =>
-            if (r.syncResult) {
-              info(s"Synchronization complete for $r")
-            }
-            else {
-              warn(s"Failed synchronizing $r")
-            }
+      Await.result(poolsService.syncOperations, 30.minutes).foreach{
```

This will throw if the timeout is reached (and other edge cases). Is it really what you want?

---

👍

---

Emm, I think I would throw the Exception than catch it.

---

That's so strange of you to say that 😂 But ok 👍